### DISCUSSION TITLE
Mirrors/Mirrors-Python

### DISCUSSION THREAD

**Discussion by giscus** [newbe36524/Newbe.Docs#258] [Announcements]
# Mirrors/Mirrors-Python

Python. 国内直接从官网 https://www.python.org 下载比较困难，需要一些技术手段。这里提供一个国内的镜像下载地址列表，方便网友下载。

https://www.newbe.pro/Mirrors/Mirrors-Python

**Comment by CCKNBC** [comment]
真不错
